### PR TITLE
[1328] Wrap courses table with tabs component

### PR DESCRIPTION
**Pull request by tvararu**
### Context

We need to present the basic details on the frontend, and the enrichment information on the ui.

### Changes proposed in this pull request

- Wrap courses table with tabs component
- "Description" tab directs to manage-courses-ui.

### Guidance to review

Review without whitespace changes.

### After

Screenshot 2019-04-17 at 18 49 42

### CODE REVIEW

**Comment by tvararu** [comment]
[Course details tabs (frontend)](https://trello.com/c/JH3J5oLU/1329-course-details-tabs-frontend)

File: app/views/courses/show.html.erb [hunk 1]
```diff
@@ -23,134 +23,161 @@
...
-        <dt class="govuk-summary-list__key">
-          Outcome
-        </dt>
-        <dd class="govuk-summary-list__value" data-qa="course__qualifications">
-          <%= course_outcome(@course) %>
-        </dd>
-      </div>
+  <ul class="govuk-tabs__list" role="tablist">
+    <li class="govuk-tabs__list-item" role="presentation">
+      <a class="govuk-tabs__tab"
```
Comment (dankmitchell) [inline app/views/courses/show.html.erb]: Can we use rails anchor tags, please

File: app/webpacker/stylesheets/_course-tabs.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.course-tabs {
+  .govuk-tabs__panel {
+    border: none;
+    padding: 30px 0 0;
+  }
+
+  .govuk-tabs__tab {
+    font-weight: bold;
+    color: #005ea5;
```
Comment (dankmitchell) [inline app/webpacker/stylesheets/_course-tabs.scss]: Colour variables?

File: app/webpacker/stylesheets/_course-tabs.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+.course-tabs {
+  .govuk-tabs__panel {
+    border: none;
+    padding: 30px 0 0;
```
Comment (dankmitchell) [inline app/webpacker/stylesheets/_course-tabs.scss]: Spacing mixin from govuk-frontend ?

**Pull request merged**